test/sim/moves/maxguard.js

```diff
@@ -34,7 +34,7 @@ describe('Max Guard', function () {
 		assert.false.fainted(minun);
 	});
 
-	it.skip('should block certain moves that bypass Protect', function () {
+	it('should block certain moves that bypass Protect', function () {
```

Comment:
I think it would make more sense to merge more of the tests into this one. It seems a bit silly to have a test just for Gear Up when this test already has a practically identical Magnetic Flux test. Surely we could just give the Pokémon in this test a few more moves to use? It’s not like they have to be legal, either. We can easily have a Kartana with Plus and like 5 moves.